code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/buildscript/OpendRootBuildScript.xtend

```diff
@@ -0,0 +1,96 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.buildscript
+
+import de.fhdo.lemma.model_processing.code_generation.container_base.template.BuildScriptTemplate
+import java.util.Map
+import java.util.HashMap
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.io.File
+import java.nio.charset.StandardCharsets
+import java.util.regex.Pattern
+
+/**
+ * This class contains the logic for the generation of the build scripts, used in the
+ * [BuildScriptCodeGenerator].
+ *
+ * @author [Philip Wizenty](mailto:philip.wizenty@fh-dortmund.de)
+ */
+class OpendRootBuildScript {
+    static OpendRootBuildScript instance
+    Map<String, String> buildScriptNodes
+
+    new () {
+        this.buildScriptNodes = new HashMap
```

Comment:
Use Xtend's `newHashMap'' command.

Reply:
Moreover, this is a trivial constructor. You may remove it and instead initialize the `buildScriptNodes` attribute directly upon its definition, i.e., `(val) buildScriptNodes = <String, String>newHashMap`.

Reply:
Please also check other singletons for trivial constructors.